docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -463,7 +468,8 @@ See the `content_collector README <https://github.com/ansible/content_collector>
 BOTMETA.yml
 -----------
 
-The `BOTMETA.yml <https://github.com/ansible/ansible/blob/devel/.github/BOTMETA.yml>`_ is the source of truth for:
+The `BOTMETA.yml <https://github.com/ansible/ansible/blob/devel/.github/BOTMETA.yml>`_ in the Ansible main repository is the source of truth for:
```

Comment:
Thank you for updating this as we now have botmeta in some of the collection repos as well.

@acozine how are we referring to the ansible/ansible repo? "ansible-base repo"?

Reply:
I've gone with "the ansible/ansible GitHub repo" for now. We can revisit, but at least the phrase is unambiguous.